tests/v2/unit/adapters/datumaro/adapter/test_anomaly_adapter.py

```diff
@@ -51,8 +51,10 @@ def test_init(self) -> None:
         assert Subset.TESTING in self.test_dataset_adapter.dataset
 
     def test_get_otx_dataset(self) -> None:
-        assert isinstance(self.train_dataset_adapter.get_otx_dataset(), DatasetEntity)
-        assert isinstance(self.test_dataset_adapter.get_otx_dataset(), DatasetEntity)
+        assert isinstance(self.train_dataset_adapter.get_label_schema(), LabelSchemaEntity)
+        assert isinstance(self.train_dataset_adapter.get_otx_dataset(), Dict)
```

Comment:
Can we use dict instead?
```suggestion
        assert isinstance(self.train_dataset_adapter.get_otx_dataset(), Dict)
```

Reply:
fixed at c865c3bbf6de80842f54f20521b95508adbce851.